actors/user/src/main/java/org/sunbird/user/dao/impl/UserDaoFactory.java

```diff
@@ -0,0 +1,30 @@
+package org.sunbird.user.dao.impl;
+
+import org.sunbird.user.dao.UserDao;
+
+/**
+ * Factory of UserDao.
+ * 
+ * @author Amit Kumar
+ *
+ */
+public class UserDaoFactory {
```

Comment:
Remove factory. If required to create instance explore using dependency injection.